### ISSUE TITLE
Which data is used under the synthetic room dataset

### ISSUE THREAD

**Issue by bearprin** [tangjiapeng/SA-ConvONet#6]
Hi, thanks for your work.

I have noticed that you randomly picked 50 shapes for each set (train/val/test) under the synthetic room dataset.

Would you like to share which data you have used (e.g., index)? I want to do the same exp as you.

Looking forward to your reply.

**Comment by tangjiapeng** [comment]
Hi, 
for train/val sets, we use the same models for pre-training.
for the test-time optimization, we use 10 scenes of test set for each category of synthetic room dataset.

room_04:
00000820
00000826
00000827
00000830
00000831
00000832
00000839
00000840
00000988
00000994

room_05:
00000805
00000816
00000838
00000839
00000856
00000869
00000870
00000919
00000990
00000998

room_06:
00000899
00000930
00000931
00000943
00000944
00000945
00000946
00000955
00000960
00000968

room_07:
00000804
00000816
00000831
00000854
00000876
00000917
00000934
00000975
00000989
00000993

room_08:
00000806
00000809
00000810
00000938
00000939
00000940
00000949
00000974
00000981
00000994

**Comment by bearprin** [comment]
Awesome! Thanks for your reply.
